s4e-backend/src/main/java/pl/cyfronet/s4e/bean/PRGOverlay.java

```diff
@@ -0,0 +1,13 @@
+package pl.cyfronet.s4e.bean;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Data
+public class PRGOverlay extends Overlay {
```

Comment:
There is one thing missing in the DB diagram that I realized is necessary recently: the SLD style of the layer.
It has to be customized from the GeoServer defaults and uploaded to the server independently.
An entity class should look like below. `name` for identification in GeoServer and `sldContent` for the xml style that has to be uploaded to the server.
```java
public class SldStyle {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String sldContent;
}
```
Then, `PRGOverlay` will need a field:
```java
    @ManyToOne
    private SldStyle sldStyle;
```